### ISSUE TITLE
Ubuntu 16.10: Alacritty.desktop opens unstyled separate icon on launcher

### ISSUE THREAD

**Issue by brycefisher** [jwilm/alacritty#330]
## Goal

I want to add Alacritty as a pinned icon on my launcher using a custom logo.

## Background

I'm on Ubuntu 16.10, and I've add copied this Alacritty.desktop to my `~/.local/share/applications/`:

```
[Desktop Entry]
Type=Application
TryExec=/usr/local/bin/alacritty
Exec=/usr/local/bin/alacritty
Icon=custom-logo
Terminal=false
X-Catagories=System;TerminalEmulator
Name=Alacritty
GenericName=Terminal
Comment=A cross-platform, GPU enhanced terminal emulator
```

I've also copied `custom-logo.png` to `/usr/share/pixmaps/`

I copied the Alacritty.desktop icon to the launcher, and the custom logo appears in the launcher as expected.

## Problem

Cicking on the logo causes a separate entry to appear on the launcher.  See screenshots:

![before](https://cloud.githubusercontent.com/assets/220106/22157399/680438ca-deec-11e6-9eeb-12f46a3286ec.png)

![after](https://cloud.githubusercontent.com/assets/220106/22157408/6e4a2bf4-deec-11e6-9d87-ce042a5843f9.png)

**Comment by tomterl** [comment]
This might be related to #348

**Comment by brycefisher** [comment]
@tomterl unrelated to #348. 

This is should be closed since its now a duplicate of #455 -- which also contains the solution to this issue.
